bindings/nodejs/examples/client/08-data-block.ts

```diff
@@ -38,9 +38,12 @@ async function run() {
         const fetchedBlock = await client.getBlock(blockIdAndBlock[0]);
         console.log('Block data: ', fetchedBlock);
 
-        if (fetchedBlock.payload instanceof TaggedDataPayload) {
-            const payload = fetchedBlock.payload as TaggedDataPayload;
-            console.log('Decoded data:', hexToUtf8(payload.data));
+        if (fetchedBlock.isBasic()) {
```

Comment:
Is that you, Rust?

Reply:
yea figured why the heck not make it easy to use ;)

Reply:
not rusty enough tho :joy: ? wen `match fetchedBlock`?